zhazam/Modules/StartGameTimer/StartGameTimerViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  StartGameTimerViewController.swift
+//  zhazam
+//
+//  Created by Abai Kalikov on 4/22/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class StartGameTimerViewController: UIViewController {
+    
+    private enum Constants {
+        static let counterArray: [String] = ["3", "2", "1", ""]
+        static let lastRowDisplayIndex = 1
+        static let pickerViewRowHeight: CGFloat = 200
+        static let reusingViewFontSize: CGFloat = 140
+        static let timerTimeInterval: TimeInterval = 1.0
+    }
+    
+    @IBOutlet weak var timerPickerView: UIPickerView!
```

Comment:
Why outlet is weak?

Reply:
Read it and understand the main usage of weak and strong refs.